### PR TITLE
Create tutorial on machine learning

### PR DESCRIPTION
**Pull request by Adimote**

### CODE REVIEW

**Review by prophile — APPROVED** [review]

File: tutorials/machine_learning.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+---
+Using machine learning
+===================
+
+in short:
+
+Don't.
+===================
+
+thanks.
```
Comment (RealOrangeOne) [inline tutorials/machine_learning.md:13]:
```suggestion
Thank you for coming to my TED talk.
```

File: tutorials/machine_learning.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+layout: page
+title: using Machine Learning
```
Comment (PeterJCLaw) [inline tutorials/machine_learning.md:3]:
```suggestion
title: Using Machine Learning
```

**Review by trickeydan — APPROVED** [review]
:shipit:

**Comment by raccube** [comment]
Please rebase this on `master`

**Comment by PeterJCLaw** [comment]
This was fun while it lasted.

**Pull request closed without merging**